main.tf

```
@@ -1,14 +1,70 @@
 data "aws_caller_identity" "default" {}
 
+module "sns_kms_key_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+  count   = local.create_kms_key ? 1 : 0
+
+  attributes = ["sns-kms-key"]
```

```suggestion
  attributes = ["sns-kms-key"]
```
If the resource is already KMS, adding `-kms-key` doesn't add further disambiguation. Are you seeing this pattern elsewhere? (if so, we can continue, but generally discourage it)

e.g. we don't need to call our s3 buckets, "my-images-bucket", it's enough to call it `my-images` since `bucket` is implied by the resource type.